.github/workflows/release.yml

```diff
@@ -16,11 +16,8 @@ jobs:
       - name: LT release
         id: lt_release
         with:
-          distribute: true
           publish_to_binaries: true
-          attach_artifacts_to_github_release: true
-          run_rules_cov: true
-          slack_channel: team-lang-php
+          slack_channel: team-lang-phpy-notify
```

Comment:
I'd suggest calling the channel `team-lang-phpy-notifs` for consistency with how other similar channels are named.